islandora_paged_content_pdf_batch.drush.inc

```diff
@@ -69,7 +65,7 @@ function islandora_paged_content_pdf_batch_drush_command() {
       ),
       'namespace' => array(
         'description' => dt('The namespace for objects created by this command.  Defaults to namespace set in Fedora config.'),
-        'required' => FALSE,
+        'required' => TRUE,
```

Comment:
It may be just my local setup.  If I didn't specify a namespace, it would crash drush (E.g., no command drush islandora_paged_content_pdf_batch.  Please drush cache-clear drush, etc).

It may not be a general problem.  On the other hand, it didn't seem too much an inconvenience to require the user to type "--namespace:something" regardless.  

It's totally understandable whether you reject or accept that line.